homeassistant/components/cover/tahoma.py

```diff
@@ -14,6 +14,15 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+ATTR_CLOSURE = 'closure'
```

Comment:
Please limit PRs to 1 feature per PR next time. So changes to binary sensor and cover: 2 prs.

Reply:
I will do so in the future. Sorry.